pkg/graveler/ref/manager.go

```diff
@@ -646,15 +646,37 @@ func (m *Manager) DeleteExpiredImports(ctx context.Context, repository *graveler
 }
 
 // Pull Requests logic
+// TODO (niro): In the future we would probably like to move all the PR logic into a dedicated service similar to actions.
+// TODO (niro): For now we put all the logic here under a single block
```

Comment:
Cool.

Not sure if it's GO way, but maybe extract this now into a separate file, to make this distinction more clear?

Reply:
I'd like to defer the decision here. Hopefully this feature will get a lot of interest and will be out of MVP in the near future and then we'll take it out of graveler altogether